test.md

```diff
@@ -10,6 +10,27 @@ module WASM-TEST
     imports WASM
 ```
 
+Helper Functions
+----------------
+
+`init_global` is a helper function that helps us to declare a new global variable.
+
+```k
+    syntax Instr ::= "init_global" Int Int
+ // --------------------------------------
+    rule <k> init_global INDEX GADDR => . ... </k>
+         <globalAddrs> GADDRS => GADDRS [ INDEX <- GADDR ] </globalAddrs>
+         <globals>
+           ( .Bag =>
+             <globalInst>
+               <gAddr> GADDR </gAddr>
+               ...
+             </globalInst>
+           )
+           ...
+         </globals>
+```
+
```

Comment:
Move this closer to where the code for `#assertGlobal` is. In the same section.

You can give `Globals` their own section (and separate them from `Locals`).